src/v1/sources/shopify/pixelTransform.js

```diff
@@ -21,6 +25,32 @@ const NO_OPERATION_SUCCESS = {
   statusCode: 200,
 };
 
+const handleCartTokenRedisOperations = async (inputEvent, clientId) => {
+  try {
+    const cartTokenLocation = eventToCartTokenLocationMapping[inputEvent.name];
+    if (!cartTokenLocation) {
+      logger.info(`Cart token location not found for event: ${inputEvent.name}`);
+      return;
+    }
+    const cartToken = _.get(inputEvent, cartTokenLocation);
```

Comment:
what about parsing the field ? cartToken in client side events will be part of the path

Reply:
Missed adding the change, have updated now, thanks for pointing it out